src/Layout/redist/minimumMSBuildVersion

```diff
@@ -1 +1 @@
-17.4.0
+17.4.6
```

Comment:
Don't do this, we never shipped 17.4.6 to VS as far as I can tell. It should be safe to leave at .0 for its stated purpose, though that'll make bumping the ref/shipped version in this repo more annoying.